sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_items_daily_v1/query.sql

```diff
@@ -0,0 +1,112 @@
+WITH events_dedupe AS (
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shared-prod.firefox_desktop_stable.newtab_v1`
```

Comment:
Shouldn't the data in `_stable` already be deduplicated as part of the `copy_deduplicate` run?

Reply:
You are correct. I do not see any duplicates for `document_id`. I will remove the filter. Thanks!